py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -1647,9 +1647,128 @@ def aten_ops_logical_xor(
     )
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.eq.Tensor)
-@dynamo_tensorrt_converter(torch.ops.aten.eq.Scalar)
-def aten_ops_equal(
+def bitwise_type_validator(node: Node) -> bool:
+    targets = [
+        torch.ops.aten.bitwise_and.Tensor,
+        torch.ops.aten.bitwise_or.Tensor,
+        torch.ops.aten.bitwise_xor.Tensor,
+    ]
+    if node.target not in targets:
+        return False
+
+    lhs_val = node.args[0]
+    rhs_val = node.args[1]
+    lhs_meta = lhs_val.meta.get("tensor_meta")
+    rhs_meta = rhs_val.meta.get("tensor_meta")
```

Comment:
If `rhs` is a Scalar, does this error out?

Reply:
I added support for Scalar with bool type, and related tests.